app/controllers/reviewables_controller.rb

```diff
@@ -69,6 +69,14 @@ def index
     render_json_dump(json, rest_serializer: true)
   end
 
+  def lightweight_index
```

Comment:
```suggestion
  def notifications_list
```

How about the above suggestion? I feel like `lightweight` is very generic and doesn't communicate the intend of the controller's action well enough.